index.js

```diff
@@ -7,23 +7,23 @@
  * Code distributed by Google as part of the polymer project is also
  * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
  */
- /*jslint node: true */
-'use strict';
 
+'use strict';
+require("babel-polyfill");
 /**
  * Static analysis for Polymer.
  * @namespace hydrolysis
  */
-module.exports = {
-  Analyzer:          require('./lib/analyzer'),
-  docs:              require('./lib/ast-utils/docs'),
-  FSResolver:        require('./lib/loader/fs-resolver'),
-  jsdoc:             require('./lib/ast-utils/jsdoc'),
-  Loader:            require('./lib/loader/file-loader'),
-  NoopResolver:      require('./lib/loader/noop-resolver'),
-  RedirectResolver:  require('./lib/loader/redirect-resolver'),
-  XHRResolver:       require('./lib/loader/xhr-resolver'),
-  StringResolver:    require('./lib/loader/string-resolver'),
-  _jsParse:          require('./lib/ast-utils/js-parse'),
-  _importParse:      require('./lib/ast-utils/import-parse'),
-};
+exports.Analyzer = require('./lib/analyzer').Analyzer;
```

Comment:
Because this file is super simple, and it's not in the src/ dir, so the choice was either just make it a js file or special case it in the build rules. I chose the simpler option